### PR TITLE
Users/tolee/private workbooks deprecation

### PR DESCRIPTION
**Pull request by tonykslee**
Adding documentation for private workbooks deprecation

### CODE REVIEW

File: Documentation/DataSources/PrivateWorkbooksDeprecation.md [hunk 1]
```diff
@@ -1,10 +1,75 @@
...
 
-One of the easiest ways is to identify a private workbook and shared workbook by their icon on the gallery view. The private workbook will have a blue icon with a person figure in the corner. The shared workbook will have a green icon with a stack figure in the corner.
+Private workbooks will be deprecated by **June 30, 2021** to simplify the consumption and sharing experience. There are two stages to private workbook deprecation.  
+
+1. First stage - Switch default save mode for workbooks to shared 
+    - Every new workbook is **now** saved as shared workbook by default. 
+    - We have removed the ability to create/ update private workbooks. 
+ 
+2. Second stage - Convert existing private workbooks to shared workbooks  
+    - <ins>**You**</ins> will need to convert your existing private workbooks to shared workbooks by
```
Comment (gardnerjr) [inline Documentation/DataSources/PrivateWorkbooksDeprecation.md:23]: what's this `ins` html tag for?

File: Documentation/DataSources/PrivateWorkbooksDeprecation.md [hunk 2]
```diff
@@ -1,10 +1,75 @@
...
+        - Either using ‘save’ / ‘save as’ (make a copy)
```
Comment (gardnerjr) [inline Documentation/DataSources/PrivateWorkbooksDeprecation.md:24]: nit: a bunch of this is using office "smart quotes" instead of just standard quotes

**Review by gardnerjr — APPROVED** [review]
approved with minor nits/suggestions

**Comment by yves-chan** [comment]
image
for some reason, the second copylinks2 looks like this, instead of an image

File: Documentation/DataSources/PrivateWorkbooksDeprecation.md (hunk 1 above)
Comment (tonykslee) [reply Documentation/DataSources/PrivateWorkbooksDeprecation.md:23]: underline

**Pull request merged**